### PR TITLE
connects Falkonry/tracker#240

### PR DESCRIPTION
**Pull request by Jaipreet95**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+pyperclip==1.5.32
```
Comment (phagunbaya) [inline requirements.txt]: Do we need this ?

File: falkonry.py [hunk 2]
```diff
@@ -802,9 +803,15 @@ def print_datastream_details(datastream_str):
     print_info("Create Time : " + (str(datetime.datetime.fromtimestamp(datastream['createTime']/1000.0))))
     print_info("Update Time : " + (str(datetime.datetime.fromtimestamp(datastream['updateTime']/1000.0))))
     print_info("Events # : " + str(datastream['stats']['events']))
+    # print_info(datastream['timePrecision'])
     if datastream['stats']['events'] > 0:
-        print_info("Events Start Time : " + (str(datetime.datetime.fromtimestamp(datastream['stats']['earliestDataPoint']/1000.0))))
-        print_info("Events End Time : " + (str(datetime.datetime.fromtimestamp(datastream['stats']['latestDataPoint']/1000.0))))
+        if datastream['timePrecision'] != 'micro':
```
Comment (phagunbaya) [inline falkonry.py]: Does this fixes this issue https://github.com/Falkonry/tracker/issues/1163 ?

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 cmd2==0.7.7
```
Comment (phagunbaya) [inline requirements.txt]: Can you upgrade this module to latest version ? Some of it's internal dependencies are deprecated now.

File: falkonry.py (hunk 2 above)
Comment (Jaipreet95) [reply falkonry.py]: This issue has nothing to do with the python client/adk
datetime.datetime.fromtimestamp() can handle -ve values as well so dates before 1970 are also handled by default
It depends on the value which is sent as the response

**Pull request merged**